packages/guides/tests/unit/Compiler/NodeTransformers/MoveAnchorTransformerTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Compiler\NodeTransformers;
+
+use phpDocumentor\Guides\Compiler\CompilerContext;
+use phpDocumentor\Guides\Compiler\DocumentNodeTraverser;
+use phpDocumentor\Guides\Nodes\AnchorNode;
+use phpDocumentor\Guides\Nodes\DocumentNode;
+use phpDocumentor\Guides\Nodes\InlineCompoundNode;
+use phpDocumentor\Guides\Nodes\ProjectNode;
+use phpDocumentor\Guides\Nodes\SectionNode;
+use phpDocumentor\Guides\Nodes\TitleNode;
+use PHPUnit\Framework\TestCase;
+
+final class MoveAnchorTransformerTest extends TestCase
+{
+    private DocumentNodeTraverser $documentNodeTraverser;
+
+    protected function setUp(): void
+    {
+        $this->documentNodeTraverser = new DocumentNodeTraverser(new class implements NodeTransformerFactory {
+            /** @return iterable<MoveAnchorTransformer> */
+            public function getTransformers(): iterable
+            {
+                //phpstan:ignore-next-line
+                yield new MoveAnchorTransformer();
+            }
+
+            /** @return array<string, int> */
+            public function getPriorities(): array
+            {
+                return [];
+            }
+        }, 30000);
```

Comment:
```suggestion
        }, 30_000);
```

Reply:
Thanks, I added this as a style rule to php-code-sniffer